### PR TITLE
feat(admin-ui): Tabs component

### PR DESCRIPTION
**Pull request by leopuleo**
## Changes
This PR adds the `Tabs` component to the `@webiny/admin-ui` package. 

We are exporting a props-based flat component based on the following [radix component](https://www.radix-ui.com/primitives/docs/components/tabs). 

The new component is fully compatible with the old `Tabs` component and it needs some refactoring when UI tokens are defined.

### CODE REVIEW

File: packages/admin-ui/src/Tabs/Tabs.tsx [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+                ))}
+            </TabsRoot>
+        );
+    }
+);
+TabsBase.displayName = TabsPrimitive.Root.displayName;
+
+const Tabs = makeDecoratable("Tabs", TabsBase);
+
+export { Tabs, TabsProps, TabsContent, TabsContentProps, TabsTrigger, TabsTriggerProps };
```
Comment (adrians5j) [inline packages/admin-ui/src/Tabs/Tabs.tsx]:
```suggestion
export { Tabs, TabsProps, TabsContent, TabsContentProps, TabsTrigger, type TabsTriggerProps };
```

**Review by adrians5j — CHANGES_REQUESTED** [review]
Quick comment.

File: packages/admin-ui/src/Tabs/Tabs.tsx (hunk 1 above)
Comment (leopuleo) [reply packages/admin-ui/src/Tabs/Tabs.tsx]: 🙏

File: packages/admin-ui/src/Tabs/Tabs.tsx [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+        return (
+            <TabsRoot ref={ref} defaultValue={defaultValue} {...props}>
+                <TabsList>
+                    {triggers.map(trigger => (
+                        <React.Fragment key={`tab-trigger-${trigger.key}`}>
+                            {React.cloneElement(trigger, { size })}
+                        </React.Fragment>
+                    ))}
+                </TabsList>
```
Comment (adrians5j) [inline packages/admin-ui/src/Tabs/Tabs.tsx]: Nit pick, but could be simplifed. Same could be done for `contents.map` code below.

```suggestion
                <TabsList>
                    {triggers.map(trigger =>
                        React.cloneElement(trigger, { size, key: `tab-trigger-${trigger.key}` })
                    )}
                </TabsList>
```

**Review by adrians5j — APPROVED** [review]
Minor nitpick, but I'd say we can merge.

**Pull request merged**